docs/source/gbd2017_models/causes/covid_19/index.rst.rst

```diff
@@ -0,0 +1,265 @@
+  * Contact tracing of exposed individuals
+  * Self-protection measures such as wearing a mask
+
+Assumptions and Limitations
++++++++++++++++++++++++++++
+
+Assumptions: 
+
+- No birth prevalence of covid-19
+- Individuals are infectious only when they occupy the I (infected) disease state
+- Self-quarentined individuals will not be infected
+- Random mixing of the population
+
+.. note::
+
+  The last two assumptions may be revisited by incorporating additional complexity
+
+Cause Model Diagram
++++++++++++++++++++
+
+S -> E -> I -> R
+
+.. list-table:: State Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Susceptible to covid-19
+   * - E
+     - Exposed
+     - Exposed to covid-19, but not yet infected
+   * - I
+     - Infected
+     - Infected with covid-19; symptomatic OR asymptomatic
+   * - R
+     - Recovered
+     - Recovered from covid-19, no longer susceptible
+
+State and Transition Data Tables
+++++++++++++++++++++++++++++++++
+
+Indicator variables:
+
+.. list-table:: Indicator Variable Definitions
+   :widths: 5 5 20
+   :header-rows: 1
+
+   * - Variable
+     - State Name
+     - Definition
+   * - symptomatic
+     - 1 = symptomatic
+       
+       0 = asymptomatic
+     - An individual is expressing covid-19 symptoms
+   * - self_quarentine
+     - 1 = self quarentine
+
+       0 = no self quarentine
+     - An individual is self isolating in their home
+   * - traced
+     - 1 = contact traced or tested
+
+       0 = not contact traced or tested
+     - An individual has been identified as exposed or infected with covid-19 by a public health official
+
+States Data
+"""""""""""
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - XXX
+     - From IHME forecasts
+   * - S
+     - excess mortality rate
+     - 0
+     - N/A
+   * - S
+     - symptomatic
+     - 0
+     - NOTE: may revisit to include flu-like symptoms from causes other than covid-19
+   * - S
+     - traced
+     - XXX
+     - This will be an experimental variable under various scenarios
+   * - E
+     - prevalence
+     - XXX
+     - From IHME forecasts
+   * - E
+     - excess mortality rate
+     - 0
+     - N/A
+   * - E
+     - symptomatic
+     - 0
+     - NOTE: may revisit to include flu-like symptoms from causes other than covid-19
+   * - E
+     - traced
+     - XXX
+     - This will be an experimental variable under various scenarios
+   * - I
+     - prevalence
+     - XXX
+     - From IHME forecasts
+   * - I
+     - excess mortality rate
+     - XXX
+     - Age-dependent... use IHME data source
+   * - I
+     - traced
+     - XXX
+     - This will be an experimental variable under various scenarios
+   * - I 
+     - symptomatic
+     - XXX
+     - Age-dependent... does IHME have data source?
```

Comment:
Maybe we can derive this from the age pattern of severe symptoms, by assuming the fraction of people in age group _a_ with any symptoms is similar to the fraction of people in that age group who end up in the ICU among those with symptoms severe enough to seek care.  I was hoping that by writing that out, I would come up with some justification for it...